acs-integration-tests/src/test/java/com/ge/predix/test/utils/ACSRestTemplateFactory.java

```diff
@@ -233,6 +253,12 @@ public OAuth2RestTemplate getACSTemplateWithNoAcsScope() {
         return new OAuth2RestTemplate(getUserWithoutAnyAcsScope());
     }
 
+    public void debugRestTemplate(final OAuth2RestTemplate restTemplate) {
```

Comment:
Wire debugging does not show the client ID and client secret or the token URI that was used to get the token. It will just say 401 with no message if it was a failure. I can remove this though as I don't think it will be necessary.